verbsense/pom.xml

```diff
@@ -0,0 +1,173 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>edu.illinois.cs.cogcomp</groupId>
+	<artifactId>illinois-verbsense</artifactId>
+	<packaging>jar</packaging>
+	<version>1.2</version>
+
+	<url>http://cogcomp.cs.illinois.edu</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+	</properties>
+	
+  <repositories>
+		<repository>
+			<id>CogcompSoftware</id>
+			<name>CogcompSoftware</name>
+			<url>http://cogcomp.cs.illinois.edu/m2repo/</url>
+		</repository>
+	</repositories>
+
+	<dependencies>
+        <!--Include the pre-trained model for running VerbSenseLabeler without training-->
+        <!--<dependency>-->
+            <!--<groupId>edu.illinois.cs.cogcomp</groupId>-->
+            <!--<artifactId>illinois-verbsense-model</artifactId>-->
+            <!--<version>1.1</version>-->
+        <!--</dependency>-->
+
+		<!--The Illinois pipeline can be used instead of the Curator for pre-processing -->
+		<dependency>
+			<groupId>edu.illinois.cs.cogcomp</groupId>
+			<artifactId>illinois-nlp-pipeline</artifactId>
+			<version>0.0.9</version>
+		</dependency>
+
+		<dependency>
+			<groupId>com.gurobi</groupId>
+			<artifactId>gurobi</artifactId>
+			<version>6.0</version>
+		</dependency>
+
+		<dependency>
+			<groupId>edu.illinois.cs.cogcomp</groupId>
+			<artifactId>illinois-common-resources</artifactId>
+			<classifier>illinoisSRL</classifier>
+			<version>1.4</version>
+		</dependency>
```

Comment:
As of two weeks ago we have broken resources into smaller chunks instead of a big resource file. 
See which resource is being used in this project, and add that directly as the dependency.